test/tests/cht/multi_cht/conj_ht.usr

```diff
@@ -0,0 +1,73 @@
+c-----------------------------------------------------------------------
+c   
+c   The domain consists of two plates of finite thickness (h=0.5) 
+c   with plane Poiseiulle flow moving between the plates from left 
+c   to right (x=0 to 8).
+c   
+c   The gap height is 1.0 (y=0 to 1).
+c   
+c   The flow inlet temperature is T=0 and the plates are heated
+c   with volumetric heat source, qvol = 1.
+c
+c   Insulated boundary conditions are applied on the solid
+c   surfaces that are not interior to the computational domain,
+c   so heat leaves only through advection at the flow outlet.
+c
+c-----------------------------------------------------------------------
+      subroutine userchk
+
+      return
+      end
+c-----------------------------------------------------------------------
+      subroutine useric (ix,iy,iz,ieg)
+
+C     Set initial conditions
+
+      include 'SIZE'
+      include 'TOTAL'
+      include 'NEKUSE'
+
+      ux   = 4.0*y*(1. - y)
+      uy   = 0.0
+      uz   = 0.0
+      temp = 0.0
+
```

Comment:
If we're setting the initial conditions from `UDF_Setup` of the `udf` file, we don't need/want to use the `useric` in the `.usr` file. I think the `udf` is just overwriting the `useric`'s initial field conditions, so `useric` in unneeded for this test.

Reply:
Yes, good point!